testdata/expected/src/github.com/sgtest/go-misc/github.com/sgtest/go-misc/go_subrepo_import/GoPackage.graph.json

```diff
@@ -112,7 +112,7 @@
     {
       "DefRepo": "github.com/golang/net",
       "DefUnitType": "GoPackage",
-      "DefUnit": "golang.org/x/net/ipv6",
+      "DefUnit": "github.com/golang/net/ipv6",
```

Comment:
Actually, DefUnit is always set to Clone Path + Subdirectory Path by
srclib-go already. For non custom import path repos, this is the same as
the pkg import path, but this breaks for custom import path repos. Since
srclib-go cannot know about the custom import path of a repo in the current
setup, IMO a better standardization is to make DefUnit always be Clone Path
+ Subdir Path, so that refs will always work for both kinds of repos.
On Apr 26, 2016 10:45 AM, "Dmitri Shuralyov" <notifications@github.com>
wrote:

> In
> testdata/expected/src/github.com/sgtest/go-misc/github.com/sgtest/go-misc/go_subrepo_import/GoPackage.graph.json
> <https://github.com/sourcegraph/srclib-go/pull/86#discussion_r61131394>:
>
> > @@ -112,7 +112,7 @@
> >      {
> >        "DefRepo": "github.com/golang/net",
> >        "DefUnitType": "GoPackage",
> > -      "DefUnit": "golang.org/x/net/ipv6",
> > +      "DefUnit": "github.com/golang/net/ipv6",
>
> Is the DefUnit now defined as Repository Clone Path + Subdirectory Path?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcegraph/srclib-go/pull/86/files/06b8a8605c7ea1b8bd74b6fe6ef8a3fab34ae8f5#r61131394>
>